site/app/models/Config.php

```diff
@@ -256,9 +256,14 @@ public function loadMasterConfigs($config_path) {
         }
 
         $this->timezone = new \DateTimeZone($this->timezone);
-
         $this->base_url = rtrim($this->base_url, "/")."/";
-        $this->cgi_url = $this->base_url."cgi-bin/";
+
+        if (isset($submitty_json['cgi_url'])){
```

Comment:
Let's use `!empty(...)` for this check as by default, I think it might be a blank string.

Reply:
Yes, right.